curriculum/challenges/english/14-responsive-web-design-22/learn-css-transforms-by-building-a-penguin/619d324f5915c929f36ae91d.md

```diff
@@ -35,6 +35,13 @@ You should add a `40%` waypoint for `@keyframes wave`.
 assert([...[...new __helpers.CSSHelp(document).getCSSRules('keyframes')].find(rule => rule?.name === 'wave')?.cssRules].some(css => css?.keyText === '40%'));
 ```
 
+You should have 4 waypoints in your keyframe.
+
+```js
+assert([...[...new __helpers.CSSHelp(document).getCSSRules('keyframes')].find(rule => rule?.name === 'wave')?.cssRules].length);
```

Comment:
```suggestion
assert([...[...new __helpers.CSSHelp(document).getCSSRules('keyframes')].find(rule => rule?.name === 'wave')?.cssRules].length === 4);
```

Reply:
I will be creating  another pr to fix this issue.